lua/wire/stools/keyboard.lua

```diff
@@ -41,12 +40,20 @@ function TOOL.BuildCPanel(panel)
 			timer.Simple(0, function() languages:ChooseOptionID(curindex) end) -- This needs to be delayed or it'll set the box to show "0"
 		end
 	end
-	
-	panel:Help("When on, automatically removes the key from the buffer when the user releases it.\nWhen off, leaves all keys in the buffer until they are manually removed.\nTo manually remove a key, write any value to cell 0 to remove the first key, or write a specific ascii value to any address other than 0 to remove that specific key.")
-	panel:CheckBox("Automatic buffer clear", "wire_keyboard_autobuffer")
+	panel:Help( "The selected language is clientside. Any keyboard you use, created by any player, will use your selection. If your keyboard layout is not available in this list, you are welcome to create it and post it as a pull request on the wiremod github page." )
 
 	panel:AddControl("Numpad", {
 		Label = "#Tool.wire_keyboard.leavekey",
 		Command = "wire_keyboard_leavekey",
 	})
+	panel:Help( "This is the key used to exit a keyboard. This option is clientside. Any keyboard you use, created by any player, will use this key." )
+
+	panel:CheckBox("Lock player controls on keyboard", "wire_keyboard_sync")
+	panel:Help( "When on, 'locks' the player into the keyboard, meaning any keys they press will not move their character around. When off, they can walk around while typing. This option is serverside, and will be saved on the keyboard through duplications." )
+	
+	panel:CheckBox("Automatic buffer clear", "wire_keyboard_autobuffer")
+	panel:Help( "When on, automatically removes the key from the buffer when the user releases it.\nWhen off, leaves all keys in the buffer until they are manually removed.\nTo manually remove a key, write any value to cell 0 to remove the first key, or write a specific ascii value to any address other than 0 to remove that specific key. This option is serverside, and will be saved on the keyboard through duplications.")
+
+	panel:CheckBox("Enter key ASCII output","wire_keyboard_enterkeyascii")
```

Comment:
Sure, we could make this clearer, but there's a "Help" label directly below this which describes it in more detail